### ISSUE TITLE
Eng Spike: WWA Map library

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#1415]

**Comment by colinmurphy01** [comment]
Since this didn't make it into 4.3, I'm reprioritizing it in the backlog

**Comment by colinmurphy01** [comment]
@greg-does-weather is this a spike?

**Comment by greg-does-weather** [comment]
Yeah, it'll make a reasonable spike once we validate the need. Until then, it'll be hard to know what kind of features and support we should be looking for.

This will likely also require some chats with the GIS team to see what alert layers are available and whether we can just use what exists or if we'll have to construct the alert shapes ourselves (oh gosh please no).

**Comment by colinmurphy01** [comment]
Blocked by #1548
